spec/timezone.html

```diff
@@ -5,6 +5,98 @@
   <h1>Temporal.TimeZone Objects</h1>
   <p>A Temporal.TimeZone object is an immutable Object referencing a time zone.</p>
 
+  <emu-clause id="sec-time-zone-names">
+    <h1>Time Zone Names</h1>
+
+    <p>
+      An ECMAScript implementation must support a number of built-in time zones.
+      At a minimum, implementations must support a built-in time zone named *"UTC"*.
+      If the return value of DefaultTimeZone is different from *"UTC"*, and does not satisfy the syntax of a |TimeZoneNumericUTCOffset|, then implementations must support that time zone as a built-in time zone as well.
```

Comment:
> `and does not satisfy the syntax of a |TimeZoneNumericUTCOffset|`

I added this in the latest revision because it seems that we should allow for the possibility that DefaultTimeZone returns an offset time zone name (`"-05:00"`)

Reply:
Does CanonicalizeTimeZone allow for that? I think it does, just checking.

Reply:
It doesn't, but offset time zones aren't passed to HostCanonicalizeTimeZoneName, only named ones.

Reply:
Does it still make sense then to say that DefaultTimeZone's return value is valid and linked-to-HCTZN canonicalized?

Reply:
Yes, I guess in 402 DefaultTimeZone is expected to always be a named time zone, but in a non-402 environment it might not be.